src/components/Player.vue

```diff
@@ -292,6 +299,15 @@ export default {
         }
       }
     },
+    toggleHandRaised() {
+      if (this.session.isSpectator && this.player.id !== this.session.playerId)
+        return;
+      if (this.player.handRaised) {
+        this.updatePlayer("handRaised", false);
+      } else {
+        this.updatePlayer("handRaised", true);
+      }
```

Comment:
Minor quibble but this can be simplified to `this.updatePlayer("handRaised", !this.player.handRaised);`

Reply:
done!